editor.planx.uk/src/pages/Team.tsx

```diff
@@ -191,17 +201,18 @@ const FlowItem: React.FC<FlowItemProps> = ({
         />
       )}
       <DashboardListItem>
-        <Box pr={4}>
-          <DashboardLink href={`./${flow.slug}`} prefetch={false}>
+        <DashboardLink href={`./${flow.slug}`} prefetch={false}>
+          <Typography variant="h4" component="h2">
             {flow.name}
-          </DashboardLink>
+          </Typography>
+          <HiddenText aria-hidden="true">{flow.slug}</HiddenText>
```

Comment:
I'm not too sure how much this is really needed? I know it's helpful as a pseudo-search until we have this feature but we can still `CTRL+F` on flow names?

Reply:
Agreed, I've pulled it out. I added as I was getting frustrated when using `CTRL+F` and typing in a slug, but it's just a case of waiting for new features to be in place.